blockmanager.go

```diff
@@ -1084,14 +1116,15 @@ out:
 			switch msg := m.(type) {
 			case *newPeerMsg:
 				b.handleNewPeerMsg(candidatePeers, msg.peer)
+				b.PeerProcessed()
```

Comment:
This isn't used anywhere else and is only a single line to signal the channel.  I'd just inline it.